pkg/operator/mon/mon.go

```diff
@@ -290,6 +292,9 @@ func (c *Cluster) createService(mon *monConfig) (string, error) {
 			Selector: labels,
 		},
 	}
+	if c.HostNetwork {
+		s.Spec.ClusterIP = v1.ClusterIPNone
+	}
 
 	s, err := c.context.Clientset.CoreV1().Services(c.Namespace).Create(s)
```

Comment:
what ip address is used for the mon service when hostnetwork is enabled? Does it still make sense to use a service in front of the mon pods? Or might the podIP be stable when using the hostnetwork?

Reply:
@travisn When setting the `ClusterIP` to `v1.ClusterIPNone` and having `hostNetwork: true` the actual endpoint IP will the `hostIP`.